docs/new-to-ocaml.md

```diff
@@ -1,15 +1,15 @@
 # New to OCaml?
 
-As a backend for the OCaml compiler, Melange shares many similarities with the
-OCaml language. Nevertheless, there are some notable differences between the
-two. This documentation aims to clarify these distinctions. For features that
-Melange inherits from OCaml, readers will be directed to the main OCaml
-documentation.
-
 If you are completely new to OCaml, it is recommended to familiarize yourself
 with the language first. [Learn OCaml](https://ocaml.org/docs) is a good
 starting point. but we recommend the following tutorials from the official OCaml
 website:
 
 - [A First Hour With OCaml](https://ocaml.org/docs/first-hour)
 - [OCaml Exercises](https://ocaml.org/exercises)
+
+As a backend for the OCaml compiler, Melange shares many similarities with the
```

Comment:
Not related to this PR, but do most devs know what "backend of compiler" means?

Reply:
True, worth explaining a bit here better